common/laws/00_distribution_of_power.txt

```diff
@@ -509,8 +512,9 @@ law_universal_suffrage = {
 	modifier = {
 		country_legitimacy_govt_total_votes_add = 110
 		country_legitimacy_govt_total_clout_add = 25
-		country_voting_power_base_add = 20
+		country_voting_power_base_add = 15
```

Comment:
Is there a reason for this? Because it makes the election results and clout further from each other, but the legitimacy is still mainly from votes, so it makes looking at the government weirder for players.